### PR TITLE
removed the bugs 🐞

### PR DESCRIPTION
**Pull request by Arunscape**
:100: :heavy_check_mark:

### CODE REVIEW

**Review by aagavin — CHANGES_REQUESTED** [review]
keep bugs, plz

**Comment by reallynewbie** [comment]
I agree with keeping the bugs

**Comment by Arunscape** [comment]
Ah, I'll add the bugs back in then

File: covid/index.css [hunk 1]
```diff
@@ -1,22 +1,34 @@
...
+  font-style: italic;
 }
 
-table > tr > td:nth-child(2n),
-table > tr > td:nth-child(3n) {
-}
+
+table> tr > td:nth-child(2n), table> tr > td:nth-child(3n){
+  text-align: right;
+}
```
Comment (aagavin) [inline covid/index.css]: nit: new line at end of file

File: cookingwithrandy.html [hunk 2]
```diff
@@ -1,5 +1,37 @@
...
+            <li>Pour into gamer mouth</li>
+            <li>Respawn and return to your gaming</li>
+        </ol>
+        <img src="./assets/CoolBro.jpg" width="142px" height="200px">
+    </h3>
+
+    <p>Bonus points:  Add Randy's <a href="./assets/snackofchampions.jpg">favourite snack of choice</a></p>
+
+</body>
+</html>
```
Comment (aagavin) [inline cookingwithrandy.html]: nit: new line at end of file

File: styles/cooking.css [hunk 3]
```diff
@@ -1,17 +1,32 @@
...
 }
 ul {
+    padding: 0;
+    text-align: center;
 }
 ol {
-}
+    padding: 0;
+    list-style-position: inside;
+}
```
Comment (aagavin) [inline styles/cooking.css]: nit: new line at end of file

File: styles/style.css [hunk 4]
```diff
@@ -1,8 +1,18 @@
...
+    color: white;
+    font-size: 64px;
+    text-align: center;
 }
 
 #links {
-}
+    background: white;
+    text-align: center;
+}
```
Comment (aagavin) [inline styles/style.css]: nit: new line at end of file

**Review by aagavin — CHANGES_REQUESTED** [review]

**Review by aagavin — APPROVED** [review]

**Comment by Arunscape** [comment]
@reallynewbie looks like this is ready to merge

**Pull request merged**